src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -0,0 +1,61 @@
+PHPUnit Bridge
+==============
+
+Utilities for PHPUnit, especially user deprecation notices management.
+
+It provides the following features:
+
+- disable the garbage collector;
+- auto-register `class_exists` to load Doctrine annotations;
+- print a user deprecation notices summary at the end of the test suite.
+
+Handling user deprecation notices is sensitive to the SYMFONY_DEPRECATIONS_HELPER
+environment variable. This env var configures 3 behaviors depending on its value:
+
+- when set to `strict`, all but legacy-tagged deprecation notices will make tests
+  fail. This is the recommended mode for best forward compatibility.
+- `weak` on the contrary will make tests ignore all deprecation notices.
+  This is the recommended mode for legacy projects that must use deprecated
+  interfaces for backward compatibility reasons.
+- any other value will respect the current error reporting level.
+
+All three modes will display a summary of deprecation notices at the end of the
+test suite, split in two groups:
+
+- **Legacy** deprecation notices denote tests that explicitly test some legacy
+  interfaces. In all three modes, deprecation notices triggered in a legacy-tagged
+  test do never make a test fail. There are four ways to mark a test as legacy:
+  - make its class start with the `Legacy` prefix;
+  - make its method start with `testLegacy`;
+  - make its data provider start with `provideLegacy` or `getLegacy`;
+  - add the `@group legacy` annotation to its class or method.
+- **Remaining/Other** deprecation notices are all other (non-legacy)
+  notices, grouped by message, test class and method.
+
+Usage
+-----
+
+Add `"symfony/phpunit-bridge": ~2.7` to the `require-dev` section of your
+composer.json file (not in `require`) and update your composer dependencies.
```

Comment:
we always use "execute `composer require "symfony/phpunit-bridge" "~2.7" --dev` in the docs, I would suggest to do the same here.